test/test.mk

```diff
@@ -13,18 +13,6 @@ else
   ANSIBLE_INVENTORY = $(ANSIBLE_INVENTORY_FOLDER)/$(ANSIBLE_INVENTORY_FILE)
 endif
 
-.PHONY: test/automated/provision
```

Comment:
The examples in the root Readme should be updated : https://github.com/newrelic/infrastructure-agent/blob/NR-164383_provision_using_terraform/README.md#testing

Reply:
I updated the README.md https://github.com/newrelic/infrastructure-agent/pull/1785/commits/622acc879cf63b330c3f8978754c572e2009fedb

Let me know wdyt :)